src/meta/processors/admin/HBProcessor.cpp

```diff
@@ -43,6 +43,37 @@ void HBProcessor::process(const cpp2::HBReq& req) {
       onFinished();
       return;
     }
+
+    if (req.disk_paths_ref().has_value()) {
+      for (const auto& [spaceId, partDiskMap] : *req.disk_paths_ref()) {
+        for (const auto& [path, partSet] : partDiskMap) {
+          std::string key = MetaKeyUtils::diskPathsKey(host, spaceId, path);
```

Comment:
Maybe just overwrite the key value in meta server, you could check if anything updated in MetaClient.